Makefile.am

```diff
@@ -41,13 +41,20 @@ DISTCLEANFILES = $(PACKAGE)-$(VERSION).tar.gz \
 	configure \
 	install-sh \
 	missing \
-	Makefile.in
+	Makefile.in \
+	.bash_history \
+	mocker-*.cfg
 
 DISTCLEANDIRS = autom4te.cache \
 	engine-plugin \
 	tmp.repos \
 	build \
-	dockerbuild
+	extra \
+	.zanata-cache \
+	dockerbuild \
+	.node-gyp \
+	exported-artifacts \
+	rpmbuild
```

Comment:
I think it will be nice to add here `node_modules` with `yarn.lock` as well. WDYT?

Reply:
Not `yarn.lock` since it is part of normal source.

`node_modules` does make some sense as you could do...
```
yarn clean
yarn install
yarn build
```
...for a nice really clean build.

Reply:
(done in the current commit)